### DISCUSSION TITLE
Load in interactive playground

### DISCUSSION THREAD

**Discussion by hatemhosny** [dofy/learn-vim#28] [Show and tell]
A great resource. Thank you.

I'm the author of [LiveCodes](https://github.com/live-codes/livecodes) (a [feature-rich](https://livecodes.io/docs/features/), open-source, client-side code playground for [80+ languages/frameworks](https://livecodes.io/docs/languages/)). LiveCodes supports a Vim mode and allows importing files from GitHub.

So I linked to your repo to allow users to load the tutorial in LiveCodes and practice vim in the browser.
Please see:
https://x.com/hatem_hosny_/status/1770545743687778439?s=20

Thank you for such a great content.

**Comment by dofy** [comment]
Great！！！

LiveCodes is awesome!!!

Thank you very much for your great work!

**Comment by hatemhosny** [comment]
Thank you ❤
